packages/signers/src/types.ts

```diff
@@ -0,0 +1,26 @@
+import type { Hash, Hex, SignTypedDataParameters } from "viem";
+
+export interface SmartAccountSigner<Inner, AuthParams, UserDetails> {
```

Comment:
I wonder if this should live in `aa-core` still. if this is here, then devs will ALWAYS need to install aa-signers which shouldn't be a requirement if devs want to bring their own in the end

Reply:
hmm good point. that makes sense.

Reply:
this is actually more in line with how we did accounts